### DISCUSSION TITLE
Accessing the backend source code for exporting PDF

### DISCUSSION THREAD

**Discussion by master2it** [jgraph/drawio#4129] [Q&A] [answered]
Hi, do you know how I can access the backend for exporting PDF? 
I need to import my shapes, and when I try to export as PDF, I can't see my shapes in the PDF.
Or help me find out how can I convert the XML to PDF?
https://convert.diagrams.net/node/export

**Comment by davidjgraph** [comment, marked as answer]
https://github.com/jgraph/draw-image-export2 is the server-side part you need for PDF generation.
